### PR TITLE
Add pallet-sudo

### PR DESCRIPTION
**Pull request by imstar15**

### CODE REVIEW

File: node/src/chain_spec.rs [hunk 1]
```diff
@@ -90,6 +90,8 @@ pub fn development_config() -> ChainSpec {
 						get_collator_keys_from_seed("Bob"),
 					),
 				],
+				// Sudo account
+				get_account_id_from_seed::<sr25519::Public>("Alice"),
```
Comment (imstar15) [inline node/src/chain_spec.rs:94]: Here is a temporary Sudo account, which needs to be modified later.

File: node/src/chain_spec.rs [hunk 2]
```diff
@@ -144,6 +146,8 @@ pub fn local_testnet_config() -> ChainSpec {
...
```
Comment (imstar15) [inline node/src/chain_spec.rs:150]: Here is a temporary Sudo account, which needs to be modified later.

Comment (chrisli30) [reply node/src/chain_spec.rs:150]: Why do we need to write a seed phrase in the chain config? It doesn’t make sense.

I just need a wallet addres to be a sudo. I shouldn’t need to give you my seed phrase.

**Review by chrisli30 — CHANGES_REQUESTED** [review]

**Review by chrisli30 — CHANGES_REQUESTED** [review]

**Review by rhuttman — APPROVED** [review]
lgtm

**Review by chrisli30 — APPROVED** [review]
My questions were addressed so looking good 👍

**Pull request merged**